.prettierrc

```diff
@@ -0,0 +1,10 @@
+{
+	"trailingComma": "none",
+	"bracketSpacing": false,
+	"arrowParens": "avoid",
+	"singleQuote": true,
```

Comment:
Should we need to change `singleQuote` to `false`? As codacy once showed that we should use *double* quotes. All our codebase are also formatted so. What do you think? @kartik918

Reply:
Yep, I agree with you. (Also Codacy is dead now 🔫)

Reply:
Which reminds me, we need a new code quality bot, or at least formatting tests before a PR

Reply:
We can use our CI(Github Action) for it. What do you say? @kartik918

Reply:
Let's discuss this in an issue of it's own later. Firstly we need prettier rules to enforce formatting.